ts/components/conversation/ConversationHeader.tsx

```diff
@@ -392,12 +379,7 @@ export class ConversationHeader extends React.Component<Props> {
           </div>
           {this.renderExpirationLength()}
 
-          {!this.props.isRss && (
-            <>
-              {this.renderSearch()}
-              {this.renderAvatar()}
-            </>
-          )}
+          {!this.props.isRss && <>{this.renderAvatar()}</>}
```

Comment:
```suggestion
          {!this.props.isRss && this.renderAvatar()}
```
`<>` were redundant. This should do the same thing. Could you check if this works and also run `yarn lint-full` once done?

Reply:
1 Second